ajax/libs/overpass/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "overpass",
+  "version": "3.0.2",
+  "filename": "overpass-webfont/overpass.css",
+  "description": "Overpass open source web font family",
+  "homepage": "http://overpassfont.org/",
+  "keywords": [
+    "font",
+    "open source font",
+    "web font"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RedHatBrand/Overpass"
+  },
+  "license": "OFL-1.1",
```

Comment:
We can see the lib is under dual license through https://github.com/RedHatBrand/Overpass/tree/3.0.2#license. So let's use 
```  
"licenses": [
  "LGPL-2.1",
  "OFL-1.1"
]
```